cool_tasks.sql

```diff
@@ -0,0 +1,23 @@
+CREATE DATABASE cool_tasks;
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+CREATE EXTENSION
```

Comment:
CREATE EXTENSION this should be moved, it is result from terminal of previous command